### PR TITLE
Make “Copy invite link” button primary action

### LINKED ISSUE #873: Switch Onboarding Link – manual adding
Please make Onboarding link a button and manual add contributor a link
ion order to priotize LINK-Onboarding

### PR DESCRIPTION
**Pull request by tillprochaska**
@drjakob @FrauCsu: I decided for "Einladung kopieren" instead of just "Neues Mitglied hinzufügen" to set clear expectations what happens when you click the button.

Screenshot 2021-05-17 at 12 13 10

Closes #873

### CODE REVIEW

File: app/components/create_invite_button/create_invite_button.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
-.CreateInviteButton-success {
+.CreateInviteButton:not(.CreateInviteButton--primary) .CreateInviteButton-success {
```
Comment (roschaefer) [inline app/components/create_invite_button/create_invite_button.css:1]: ~Why is `:not(.CreateInviteButton--primary)` necessary?~ OK, understood.

File: app/components/create_invite_button/create_invite_button.css [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
   color: var(--color-green-dark);
 }
```
Comment (roschaefer) [inline app/components/create_invite_button/create_invite_button.css:3]:
```suggestion
```

File: app/views/contributors/index.html.erb [hunk 3]
```diff
@@ -7,8 +7,12 @@
       <% end %>
 
       <%= c 'stack', space: :small, class: 'text-center' do %>
-        <%= c 'button', label: I18n.t('contributor.create'), link: new_contributor_path %><br>
-        <%= c 'create_invite_button' %>
+        <%= c 'create_invite_button', style: :primary %><br>
```
Comment (roschaefer) [inline app/views/contributors/index.html.erb:10]:
```suggestion
        <%= c 'create_invite_button' %><br>
```

**Review by roschaefer — CHANGES_REQUESTED** [review]
so far so good

I'd prefer not to add specializations that we don't use.

File: app/components/create_invite_button/create_invite_button.css (hunk 1 above)
Comment (tillprochaska) [reply app/components/create_invite_button/create_invite_button.css:1]: Is this what you where referring in this comment?

> I'd prefer not to add specializations that we don't use.

File: config/locales/de.yml [hunk 4]
```diff
@@ -207,7 +203,10 @@ de:
     contributor:
       one: Mitglied
       other: Mitglieder
-    create: Neues Mitglied erstellen
+    actions:
+      create: Manuell hinzufügen
+      copy_invite: Einladung kopieren
```
Comment (roschaefer) [inline config/locales/de.yml:208]: I prefer `Einladungs-Link`.

**Review by roschaefer — CHANGES_REQUESTED** [review]
firefox

On Firefox I see this:

swappy-20210518_002724

**Comment by FrauCsu** [comment]
giphy

File: app/components/copy_button/copy_button.css [hunk 5]
```diff
@@ -1,11 +1,34 @@
...
+.CopyButton[data-state='loading'] .CopyButton-loading {
+  display: initial;
+}
+
+.CopyButton[data-state='success'] .CopyButton-success {
+  display: initial;
+}
+
+.Button:not(.Button--primary) .CopyButton-success {
+  color: var(--color-green-dark);
```
Comment (roschaefer) [inline app/components/copy_button/copy_button.css:33]: Why is this so complicated? Isn't there a way to render HTML conditionally with Stimulus? E.g. [alpine](https://github.com/alpinejs/alpine) can do this easily at runtime, without any integration issues.

**Review by roschaefer — APPROVED** [review]
fine

**Comment by roschaefer** [comment]
@tillprochaska I rebased. Feel free to merge, once the build server is ready.

**Pull request merged**